dev-manual/env/compose.rst

```diff
@@ -9,7 +9,9 @@ are developed, built, distributed and deployed. The AtoM team is experimenting
 with new workflows that make use of containers. This document introduces our
 new development workflow based on Docker and `Docker Compose <https://docs.docker.com/compose/>`__.
 The latter is a tool that help us to run multi-container applications like AtoM
-and it is suitable for both development and production environments.
+and it is suitable for both development and production environments. However,
+the following Docker Compose integration and the AtoM Dockerfile are not fully
+prepared for production environmets at this point.
```

Comment:
typo here - "environmets" - should be "environments" ;)

I also wonder if this should be put in an admonition to draw attention to it? E.g.:

```
.. _User forum: https://groups.google.com/forum/#!forum/ica-atom-users

.. WARNING:
   
   The Docker Compose integration and AtoM Dockerfile outlined here is intended for **testing and development 
   purposes only**. It has not been fully tested and prepared for use in production environments at this time. 

   Artefactual provides this documentation as-is - we do not regularly test or maintain the instructions outlined 
   here as part of our release management process. If you encounter issues or bugs, you are welcome to report them 
  via the AtoM User forum_ - we will provide basic support as we are able, but urge you to proceed at your own risk.
```